src/app/learn/reference/extensibility/index.html

```diff
@@ -16,80 +16,96 @@
+For information about SKY UX CLI commands, see <a routerLink="/learn/reference/cli-commands">the CLI commands reference</a>.
 
 ## SKY UX Builder
 
-SKY UX Builder is intended to be listed as a `devDependency` for every SPA.  The majority of commands are handled through it.  Testing, serving, and building are all functions of SKY UX Builder.  As such, it wraps up the logic and configuration for other tools such as the Angular Compiler, Webpack, Karma, and Protractor.
-
-## SKY UX CLI Plugins
-
-SKY UX CLI is extensible.  By following the guidelines below, it is possible to create custom commands and modules to handle those commands.
-
-- The module must be installed globally or locally.
-- The module's name must begin with either `skyux-builder` or `@skyux-sdk/builder`.
-- The module must specify the `main` property in its `package.json` or contain an `index.js` in the root.  One of this is necessary in order for your module to be used with `require()`.
-- The module must expose a `runCommand` method.  This method is called with two params, the requested command and an arguments object as processed by <a href="#">minimist</a>.  This method should return a boolean as to whether it successfully handled the command.
-
-As an example of whatever file you choose to satisfy the last requirement above:
-
-```
-module.exports = {
-  runCommand(command, argv) {
-    switch (command) {
-      case 'customCommand':
-        // Do something here
-        return true;
-
-      default:
-        return false;
-    }
-  }
-}
-```
-
-## SKY UX Builder Plugins
-
-Add the package as a dependency in `package.json`.
-
-```
-{
-  "dependencies": {
-    "skyux-builder-plugin-example": "1.0.0"
-  }
-}
-```
-
-In the `skyuxconfig.json` file, specify the module in the `plugins` array.  For example:
-
-```
-{
-  "plugins": [
-    "skyux-builder-plugin-example"
-  ]
-}
-```
-
-Your package must contain one of the following:
-  - The `package.json` must specify the `main` property which points to a valid JavaScript file.
-  - An `index.js` file in the root.
-
-As an example of whatever file you choose to satisfy the above requirement:
-
-```
-module.exports = {
-  // content = a string representation of the file's contents
-  // resourcePath = the absolute path to the file
-  // skyPagesConfig = the prepared config that includes skyuxconfig.json and builder's defaults
-  preload(content, resourcePath, skyPagesConfig) {
-    return content;
-  }
-}
-```
+SKY UX Builder is intended to be listed as a `devDependency` in the `package.json` file for every SPA. Builder handles the majority of `skyux` commands, including testing, serving, and building, and it wraps up the logic and configuration for other tools such as the Angular Compiler, Webpack, Karma, and Protractor.
+
+## SKY UX CLI plugins
+
+SKY UX CLI is extensible. Follow these guidelines to create custom commands and modules to handle those commands:
+
+<ul>
+  <li>
+    Install the module globally or locally.
```

Comment:
Changing the tense of this sentence highlights that this step is really geared towards the consumers.  I'm not sure if it's best to just remove it from here, or to create a new section.  This would be necessary to consume the plugin, but not required for someone creating a plugin.